doc/src/devdocs/callconv.md

```diff
@@ -1,60 +1,43 @@
-.. _devdocs-callconv:
-
-*******************
-Calling Conventions
-*******************
-
-.. currentmodule:: Base
+# Calling Conventions
 
 Julia uses three calling conventions for four distinct purposes:
 
-+-------+--------------+----------------------------------+
-|Name   |Prefix        |Purpose                           |
-+=======+==============+==================================+
-|Native |``julia_``    | Speed via specialized signatures |
-+-------+--------------+----------------------------------+
-|JL Call|``jlcall_``   | Wrapper for generic calls        |
-+       +--------------+----------------------------------+
-|       |``jl_``       | Builtins                         |
-+-------+--------------+----------------------------------+
-|C ABI  |``jlcapi_``   | Wrapper callable from C          |
-+-------+--------------+----------------------------------+
+| Name    | Prefix    | Purpose                          |
+|:------- |:--------- |:-------------------------------- |
+| Native  | `julia_`  | Speed via specialized signatures |
+| JL Call | `jlcall_` | Wrapper for generic calls        |
+| JL Call | `jl_`     | Builtins                         |
```

Comment:
Cool. In case you hadn't guessed, I've been checking a few pages using http://services.w3.org/htmldiff, looking at a branch on my fork where I pushed the last pre-conversion commit's html docs (https://github.com/tkelman/julia/commits/gh-pages-preconversion). Haven't done too many yet, but the ones I've looked at have been remarkably clean.

We do need to figure something out for the windows buildbots though.